src/test/unit_tests/jest_tests/test_nc_account_cli.test.js

```diff
@@ -603,6 +603,26 @@ describe('manage nsfs cli account flow', () => {
             const res = await exec_manage_cli(type, action, account_options);
             expect(JSON.parse(res.stdout).error.code).toBe(ManageCLIError.InvalidSupplementalGroupsList.code);
         });
+
+        it('cli account add - with account name suffix `.json`', async function() {
+            const action = ACTIONS.ADD;
+            const { type, new_buckets_path, uid, gid } = defaults;
+            await fs_utils.create_fresh_path(new_buckets_path);
+            await fs_utils.file_must_exist(new_buckets_path);
+            const name = 'account_suffix.json';
+            const allow_bucket_creation = 'true'; // root accounts manager is not allowed to create buckets
+            const account_options = { config_root, name, new_buckets_path, uid, gid,
+                allow_bucket_creation };
+            const res = await exec_manage_cli(type, action, account_options);
+            const parse_res = JSON.parse(res);
+            expect(parse_res.response.code).toEqual(ManageCLIResponse.AccountCreated.code);
+            expect(parse_res.response.reply.name).toEqual(name);
+            const account = await config_fs.get_account_by_name(name);
+            expect(account).not.toBeNull();
```

Comment:
Are you sure you want to check if the value is `null`?

Reply:
It should be fine, it won't break any code.